lib/emerald/grammar/emerald.tt

```diff
@@ -26,9 +26,18 @@ grammar Emerald
     (tag_statement / comment) newline <Line>
   end
 
-  # Rule for keyword list (metas, scripts, styles)
-  rule list
-    keyword newline lbrace newline (inline_literal_string newline)+ rbrace newline <List>
+  # Special rule for list
+  rule list_special
+    special_keyword newline lbrace newline
+    (inline_literal_string newline)+
```

Comment:
`list_items:(inline_literal_string newline)+`
so that you can refer to these as `list_items` later instead of `elements[4]`